src/Components/AddTask.jsx

```diff
@@ -0,0 +1,73 @@
+// Add task button component
```

Comment:
Could you follow the convention while naming the folder? If you are naming folders with all lowercase then stick with it.